components/tyk-dashboard/README.md

```diff
@@ -0,0 +1,91 @@
+## Tyk Dashboard
+[Tyk](https://tyk.io) is an open source Enterprise API Dashboard, supporting REST, GraphQL, TCP and gRPC protocols.
+
+Tyk Dashboard is provided ‘Batteries-included’, with no feature lockout. Enabling your organization to control who accesses your APIs, when they access, and how they access it. Tyk Dashboard can also be deployed as part of a larger Full Lifecycle API Management platform Tyk Self-Managed which also includes Management Control Plane, Dashboard GUI and Developer Portal.
+
+[Overview of Tyk Dashboard](https://tyk.io/docs/tyk-dashboard/)
+
+## Introduction
+This chart defines a standalone Tyk Dashboard component on a [Kubernetes](https://kubernetes.io/) cluster using the [Helm](https://helm.sh/) package manager.
+
+For typical usage, we recommend using following umbrella charts:
+* For Tyk Open Source, please use [tyk-oss](https://github.com/TykTechnologies/tyk-charts/tree/main/tyk-oss)
+* For Tyk Pro, please use either [tyk-mdcb-data-plane](https://github.com/TykTechnologies/tyk-charts/tree/main/tyk-mdcb-data-plane) or [tyk-single-dc](https://github.com/TykTechnologies/tyk-charts/tree/main/tyk-single-dc)
+
+[Learn more about different deployment options](https://tyk.io/docs/apim/)
+
+## Prerequisites
+* Kuberentes 1.19+
+* Helm 3+
+* [Redis](https://tyk.io/docs/planning-for-production/redis/) should already be installed or accessible by the dashboard 
+* [Mongo](https://tyk.io/docs/planning-for-production/database-settings/mongodb/) or [PostgreSQL](https://tyk.io/docs/planning-for-production/database-settings/postgresql/) should already be installed or accessible by the dashboard
+
+## Installing the Chart
+
+To install the chart from the Helm repository in namespace `tyk` with the release name `tyk-dashboard`:
+
+    helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
+    helm show values tyk-helm/tyk-dashboard > values-dashboard.yaml
+    helm install tyk-dashboard tyk-helm/tyk-dashboard -n tyk --create-namespace -f values-dashboard.yaml
+
+Note: Set redis connection details first. See [Configuration](#configuration) below.
```

Comment:
@andrei-tyk please list the mandatory configurations here

Reply:
added